x-pack/plugins/timelines/public/components/t_grid/toolbar/fields_browser/field_browser.tsx

```diff
@@ -166,7 +166,7 @@ const FieldsBrowserComponent: React.FC<Props> = ({
             </EuiFlexItem>
             <EuiFlexItem grow={false}>
               {CreateFieldButton && dataViewId != null && dataViewId.length > 0 && (
-                <CreateFieldButton onClick={onHide} />
+                <CreateFieldButton onHide={onHide} />
```

Comment:
nit: Could you please explain the usage of onHide for CreateFieldButton (add comment). My concern is that the name of the event doesn't clearly explain the action happened in the button use case flow. For example, onClick clearly says that the button was clicked, but what mean onHide for the button control.
I've missed this during previous field browser [PR](https://github.com/elastic/kibana/pull/126105) review

Reply:
Hi Yuliia, yes I changed the name because using _onClick_ we were delegating the logic of what happens when the _CreateFieldButton_ is clicked to the timeline (alerts) plugin, and I think this is not desirable.
This is more clear with the new _getFieldTableColumns_ optional parameter (security solution runtime action column buttons need to close the modal as well), it makes no sense to pass the _onClick_ callback to the _getFieldTableColumns_ and put the logic to close the modal in the timeline plugin. I think it is much better to make the _CreateFieldButton_ and  _getFieldTableColumns_ ask for an `onHide` (we could call `hideModal` as well) function, so the alert plugin just needs to know that these functions need to close the modal at some point, but the logic of how the modal is closed remains inside the security solution components. Does it make sense?